express/blocks/pricing/pricing.js

```diff
@@ -501,16 +501,13 @@ function decorateTable($block, features) {
 }
 
 async function fetchPricingSheet(sheet) {
-  const data = {};
   const tabs = ['header', 'plans', 'plan-options', 'table'];
   const names = ['header', 'plans', 'planOptions', 'features'];
-  await Promise.all(tabs.map(async (tab, i) => {
-    try {
-      data[names[i]] = await fetchPricingTab(sheet, tab);
-    } catch (e) {
-      // something went wrong
-    }
-  }));
+  const sheets = await fetchPricingTabs(sheet);
+  const data = {};
+  tabs.forEach((tab, i) => {
+    data[names[i]] = sheets[tab].data;
```

Comment:
I'd wrap this in a try/catch block in case `tab` isn't a valid sheet name.
```suggestion
    try {
      data[names[i]] = sheets[tab].data;
    } catch (e) {
      // something went wrong
    }
```